docs/sdks/android/id-capture/get-started.md

```diff
@@ -36,18 +36,15 @@ The Scandit Data Capture SDK modules depend on a few standard libraries that you
 
 If you directly add the AAR files to the project, you need to add these dependencies yourself.
 
-| Module      | Dependencies |
-| ----------- | ----------- |
-| *ScanditCaptureCore.aar*      | org.jetbrains.kotlin:kotlin-stdlib:[version]; androidx.annotation:annotation:[version]; com.squareup.okhttp3:okhttp:4.9.2       |
-| *ScanditBarcodeCapture.aar*   | org.jetbrains.kotlin:kotlin-stdlib:[version]; androidx.annotation:annotation:[version]        |
-| *ScanditParser.aar*      | No dependencies       |
-| *ScanditIdCapture.aar*      | org.jetbrains.kotlin:kotlin-stdlib:[version]; androidx.annotation:annotation:[version]       |
+* `org.jetbrains.kotlin:kotlin-stdlib:[version]`
+* `androidx.annotation:annotation:[version]`
+* `com.squareup.okhttp3:okhttp:4.9.2`
```

Comment:
Listing external deps independently for ScanditCaptureCore and ScanditIdCapture makes little sense, you anyways always have to include CaptuerCore and IdCapture so we can as well make this a flat list.

On a related note, ScanditParser and ScanditBarcodeCapture can be removed entirely here, they are not required for ID Capture anymore.